index.html

```diff
@@ -57,16 +57,25 @@ <h2>Jobs & Projects</h2>
     <div class="row">
         <div class="col-sm-12 supporters">
             <h2>Supporters</h2>
-              <img src="images/supporters/creativecommons.svg" height="40px" width="auto" alt="Creative Commons Logo"/>
-              <img src="images/supporters/discourse.svg" height="50px" width="auto" alt="Discourse Logo"/>
-              <img src="images/supporters/mozilla.svg" height="50px" width="auto" alt="Mozilla Logo"/>
-              <img src="images/supporters/nextcloud.svg" height="70px" width="auto" alt="Nextcloud Logo"/>
-              <img src="images/supporters/opentechfund.svg" height="40px" width="auto" alt="OpenTechFund Logo"/>
-              <img src="images/supporters/phplist.svg" height="50px" width="auto" alt="PhpList Logo"/>
-              <img src="images/supporters/redhat.svg" height="50px" width="auto" alt="RedHat Logo"/>
-              <a title="Stickermule" href="httsp://stickermule.com/supports/opensourcedesign19-sponsorship" target="_blank" rel="noopener"><img src="images/supporters/stickermule.svg" height="40px" width="auto" alt="Stickermule Logo"/></a>
-              <img src="images/supporters/wikimedia-germany.svg" height="50px" width="auto" alt="Wikimedia Germany Logo"/>
-              <br>
+            <a title="creativecommons" href="https://creativecommons.org/">
+              <img src="images/supporters/creativecommons.svg" height="40px" width="auto" alt="Creative Commons Logo"/></a>
+            <a title="discourse" href="https://www.discourse.org/">
+              <img src="images/supporters/discourse.svg" height="50px" width="auto" alt="Discourse Logo"/></a>
+            <a title="mozilla" href="https://www.mozilla.org/en-US/about/manifesto/">
+              <img src="images/supporters/mozilla.svg" height="50px" width="auto" alt="Mozilla Logo"/></a>
+            <a title="nextcloud" href="https://nextcloud.com/">
+              <img src="images/supporters/nextcloud.svg" height="70px" width="auto" alt="Nextcloud Logo"/></a>
+            <a title="opentechfund" href="https://www.opentech.fund/">
+              <img src="images/supporters/opentechfund.svg" height="40px" width="auto" alt="OpenTechFund Logo"/></a>
+            <a title="phplist" href="https://www.phplist.com/">
+              <img src="images/supporters/phplist.svg" height="50px" width="auto" alt="PhpList Logo"/></a>
+            <a title="redhat" href="https://www.redhat.com/en">
+              <img src="images/supporters/redhat.svg" height="50px" width="auto" alt="RedHat Logo"/></a>
+            <a href="">
+              <a title="Stickermule" href="https://www.stickermule.com/en-in" target="_blank" rel="noopener"><img src="images/supporters/stickermule.svg" height="40px" width="auto" alt="Stickermule Logo"/></a>
```

Comment:
Best just cut any subfolder or domain, in this case it sets the language which is different for everyone. :)